browser/main/Main.js

```diff
@@ -144,6 +152,28 @@ class Main extends React.Component {
     }
   }
 
+  handleFullScreenButton (e) {
+    this.setState({ fullScreen: !this.state.fullScreen }, () => {
+      const noteDetail = document.querySelector('.NoteDetail')
+      const mainBody = document.querySelector('#main-body')
+      const sliderRight = document.querySelector('#slider-right')
+      const sliderLeft = document.querySelector('#slider-left')
+      if (this.state.fullScreen) {
+        this.state.noteDetailWidth = noteDetail.style.left
+        this.state.mainBodyWidth = mainBody.style.left
+        noteDetail.style.left = '0px'
+        mainBody.style.left = '0px'
+        sliderRight.style.display = 'none'
+        sliderLeft.style.display = 'none'
+      } else {
+        noteDetail.style.left = this.state.noteDetailWidth
+        mainBody.style.left = this.state.mainBodyWidth
+        sliderRight.style.display = 'block'
+        sliderLeft.style.display = 'block'
+      }
+    })
+  }
```

Comment:
This is the last step (probably :wink:). Let's change some codes in order to easy to maintain. To begin with, you can remove variables. I assume `slideRight` and `slideLeft` is not required, only `noteDetail` and `mainBody` is needed in this method.
Next, you can separate this method into 3, `show`, `hide`, and `handleFullScreenButton`.

```
  handleFullScreenButton (e) {
    const noteDetail = document.querySelector('.NoteDetail')
    const mainBody = document.querySelector('#main-body')

    this.setState({ fullScreen: !this.state.fullScreen }, () => {
      if (this.state.fullScreen) {
        this.hideLeftLists(noteDetail, mainBody)
      } else {
        this.showLeftLists(noteDetail, mainBody)
      }
    })
  }

  hideLeftLists(noteDetail, mainBody) {
    this.state.noteDetailWidth = noteDetail.style.left
    this.state.mainBodyWidth = mainBody.style.left
    noteDetail.style.left = '0px'
    mainBody.style.left = '0px'
  }

  showLeftLists(noteDetail, mainBody) {
    noteDetail.style.left = this.state.noteDetailWidth
    mainBody.style.left = this.state.mainBodyWidth
  }
```

And I would like to have your opinion. Are the names of each method proper? I think they're not the best.